src/main/resources/de/tum/cit/ase/ares/api/templates/phobos/phobos_wrapper.sh

```
@@ -0,0 +1,115 @@
+}
+
+for rule in "${network_rules[@]}"; do
+    [[ $rule =~ ^allow[[:space:]]+(.+)$ ]] || continue
+    parse_hostport "${BASH_REMATCH[1]}" >> "$allowed_file"
+    echo "Added network rule: ${BASH_REMATCH[1]}" >&2
+done
+
+export NETBLOCKER_CONF="$allowed_file"
+export LD_PRELOAD="$core/libnetblocker.so"
+
+rlimit_arg=(); [[ $mem_mb -gt 0 ]] && rlimit_arg=( --rlimit-as=$((mem_mb*1024*1024)) )
```

Shellcheck (0.10.0)

[warning] rlimit_arg appears unused. Verify use (or export if used externally).

(SC2034)